pkg/utils/utils.go

```diff
@@ -315,6 +315,17 @@ func configSriovDevice(iface *sriovnetworkv1.Interface, ifaceStatus *sriovnetwor
 			}
 		}
 	}
+	// Set PF link up for kubernetes cluster
+	if ClusterType == ClusterTypeKubernetes && ifaceStatus.Name != "" {
+		pfLink, err := netlink.LinkByName(ifaceStatus.Name)
+		if err != nil {
```

Comment:
you can check if the pfLink is not up with 
pfLink.Attrs().OperState == OperUp or something like that. 

Also in what case ifaceStatus.Name will be ""?

Reply:
If some modules are not loaded like ipoib for IB interfaces

Reply:
Added condition for state and removed the condition for name